### ISSUE TITLE
Training the Image Embedder

### ISSUE THREAD

**Issue by Zumbalamambo** [PyTorchLightning/lightning-flash#114]
I would like to perform large scale image retrieval. How do I retrain the Image Embedder?

I have clustered the images into multiple folders by clustering based on the pose and trained the ImageClassifier thinking that I can use it as a model to extract the embedding. Unfortunately the loss is not reducing at all. Is there any specific way to train for image retrieval?

**Comment by Zumbalamambo** [comment]
+1

**Comment by edgarriba** [comment]
@Zumbalamambo do you have already a pipeline or some code to take a look at?

**Comment by Zumbalamambo** [comment]
+1 still a problem

**Comment by ethanwharris** [comment]
Hi @Zumbalamambo Could you provide some more details about the issue? If the loss isn't reducing for your classification data then it's possible that it's just not a very solvable problem. Have you been able to train effectively on this data outside of flash?
